Open Judge System/Data/OJS.Data.Models/Problem.cs

```diff
@@ -74,7 +74,7 @@ public class Problem : DeletableEntity, IOrderable
 
         public virtual ICollection<SubmissionType> SubmissionTypes { get; set; } = new HashSet<SubmissionType>();
         
-        public virtual ICollection<ProblemSubmissionTypeSkeleton> ProblemSubmissionTypesSkeletons { get; set; } = new HashSet<ProblemSubmissionTypeSkeleton>();
+        public virtual ICollection<ProblemSubmissionTypeExecutionDetails> ProblemSubmissionTypesSkeletons { get; set; } = new HashSet<ProblemSubmissionTypeExecutionDetails>();
```

Comment:
Change name ProblemSubmissionTypesSkeletons to ProblemSubmissionTypeExecutionDetails